src/Workspaces/Remote/ServiceHub/Shared/RemoteEndPoint.cs

```diff
@@ -194,7 +210,8 @@ private void OnDisconnected(object sender, JsonRpcDisconnectedEventArgs e)
             _debuggingLastDisconnectReason = e;
             _debuggingLastDisconnectCallstack = new StackTrace().ToString();
 
-            if (e.Reason != DisconnectedReason.LocallyDisposed)
+            if (e.Reason != DisconnectedReason.LocallyDisposed &&
+                e.Reason != DisconnectedReason.RemotePartyTerminated)
```

Comment:
can we doc what' going on here.?

Reply:
This is just writing info about why we got disconnected to the log. We don't want to write info in cases that are common - such as if we dispose the connection or the remote host process disappears (on shutdown). That would make the log noisy.